### ISSUE TITLE
is something wrong with my StarGan loss curve?

### ISSUE THREAD

**Issue by superhg** [yl4579/StarGANv2-VC#57]
Hi, @yl4579  here is my training loss curve and eval loss curve,  convert result is not good. The loss curve indicates overfitting?

![image](https://user-images.githubusercontent.com/6316449/178232871-6ee57f72-727c-445d-8aa8-5cdac1f22646.png)
![image](https://user-images.githubusercontent.com/6316449/178232940-3d120ed4-d4ec-4696-9241-aa6d7ab4efba.png)

**Comment by yl4579** [comment]
You may need to train for a longer time. 60 epochs are not enough for GAN to converge. Your cycle consistency loss is still decreasing.

**Comment by skol101** [comment]
@superhg I wonder why you have eval/reg and eval/con_reg values? They are not used in the evaluation step as per code.
